tools/util/ReplayInterpreter.cpp

```diff
@@ -55,20 +56,19 @@ void ReplayInterpreter::buildOperations(void)
       if (   m_json["payload"]["major"] != UMPIRE_VERSION_MAJOR
           || m_json["payload"]["minor"] != UMPIRE_VERSION_MINOR
           || m_json["payload"]["patch"] != UMPIRE_VERSION_PATCH ) {
-        std::cerr << "Warning, version mismatch:\n"
+
+        REPLAY_ERROR("Warning, version mismatch:\n"
```

Comment:
Should we attempt to replay mismatched versions? Maybe only error if the major version changes (from now on).

Reply:
Please see [UM-418](https://rzlc.llnl.gov/jira/browse/UM-418), we will address this as part of a separate PR.